conda/recipes/libcucim/meta.yaml

```diff
@@ -61,14 +61,10 @@ requirements:
     - openslide
   run:
     {% if aarch64 and cuda_major == "12" %}
```

Comment:
Do we need to retain the ARM specifier here? We want to require `cuda-version>=12.2,<13.0a0` for x64, too.